environments/conda-win10.yml

```diff
@@ -10,7 +10,7 @@ dependencies:
     - pexpect
     - pip
     - psutil
-    - python
+    - python==3.9

```

Comment:
This resolved issues building installer.  Weird, because it had been working with Python 3.10 before, but something around the Python 3.11 rollout seemed to break something.